src/Field/Email.php

```diff
@@ -25,34 +35,66 @@ class Email extends Field
      */
     public $include_names = false;
 
+    /**
+     * @var array Array of allowed separators
+     */
+    public $separator = [','];
+
+    /**
+     * Perform normalization.
+     *
+     * @param mixed $value
+     *
+     * @throws ValidationException
+     *
+     * @return mixed
+     */
     public function normalize($value)
     {
-
-        // use comma as separator
-        $emails = explode(',', $value);
+        // split value by any number of separator characters
+        $emails = preg_split('/['.implode('', array_map('preg_quote', $this->separator)).']+/', $value, -1, PREG_SPLIT_NO_EMPTY);
 
         if (!$this->allow_multiple && count($emails) > 1) {
-            throw new Exception(['Only a single email can be entered', 'email'=>$value]);
+            throw new ValidationException([$this->name => 'Only a single email can be entered']);
         }
 
-        array_map(function ($email) {
+        // now normalize each email
+        $emails = array_map(function ($email) {
             $email = trim($email);
 
             if ($this->include_names) {
                 $email = preg_replace('/^[^<]*<([^>]*)>/', '\1', $email);
             }
 
-            if (!filter_var($email, FILTER_VALIDATE_EMAIL)) {
-                throw new Exception(['Email format is invalid', 'email'=>$email]);
+            // should actually run only domain trough idn_to_ascii(), but for validation purpose this way it's fine too
```

Comment:
test-case for this?

Reply:
oh i see it!

Reply:
this comment line (69) should actually be removed now i guess :)